src/utils/copyTextToClipboard.ts

```diff
@@ -0,0 +1,10 @@
+export const copyTextToClipboard = (value: string) => {
+  const tempTextArea = document.createElement("textarea");
+  tempTextArea.value = value;
+  document.body.appendChild(tempTextArea);
+  tempTextArea.select();
+  document.execCommand("copy");
+  document.body.removeChild(tempTextArea);
+
+  alert("Texto copiado para a área de transferência!");
```

Comment:
Alert?

Change this to a toast?

u can remove the line 9 and set a toastify after copyTextToClipboard has been executed.

Reply:
done